package.json

```diff
@@ -1,67 +1,77 @@
+    "dev:manager": "yarn workspace @slicemachine/manager dev",
+    "dev:start-slicemachine": "yarn workspace start-slicemachine dev",
+    "dev:plugin-kit": "yarn workspace @slicemachine/plugin-kit dev",
+    "dev:slice-machine-ui": "yarn workspace slice-machine-ui dev",
+    "dev:adapter-next": "yarn workspace @slicemachine/adapter-next dev",
+    "dev:e2e-next": "cd ./e2e-projects/next && yarn dev --port 8000",
     "cypress-setup": "./cypress-setup.sh",
     "clean-e2e-projects": "git checkout e2e-projects/ && git clean -f e2e-projects/",
-    "prepare": "husky install",
-    "build": "npm --workspaces --if-present run build && npm --workspaces --if-present run test",
-    "build-essential": "npm --workspace @slicemachine/plugin-kit run build && npm --workspace @slicemachine/manager run build",
-    "audit": "lerna run audit",
-    "lint": "npm --workspaces --if-present run lint",
+    "postinstall": "husky install",
+    "build": "yarn workspaces foreach --topological-dev --verbose run build && yarn run test",
+    "build-essential": "yarn workspace @slicemachine/plugin-kit build && yarn workspace @slicemachine/manager build",
+    "audit": "yarn workspaces foreach --parallel --verbose run audit",
+    "lint": "yarn workspaces foreach --verbose run lint",
     "lint-staged": "lint-staged",
     "prettier:fix": "prettier --write .",
     "prettier:check": "prettier --check .",
-    "test": "lerna run test --stream",
-    "test:e2e": "npm run cypress-setup && start-server-and-test 'npm run dev --prefix e2e-projects/cypress-next-app' http://localhost:3000 'SM_ENV=development ENABLE_SENTRY=false npm run slicemachine --prefix e2e-projects/cypress-next-app' http://localhost:9999 'cypress run'",
-    "test:e2e:dev": "npm run cypress-setup && start-server-and-test 'npm run dev --prefix e2e-projects/cypress-next-app' http://localhost:3000 'SM_ENV=development ENABLE_SENTRY=false npm run slicemachine --prefix e2e-projects/cypress-next-app' http://localhost:9999 'cypress open'",
-    "::changeset": "npm run changeset add && git add .changeset && git commit -m '[release:changelog] update changeset'; echo 'We commited your changeset. You should defo push this master ✌️'",
-    "::release": "manypkg check && npm run build && lerna publish && npm run run changeset",
+    "test": "yarn workspaces foreach --parallel --topological-dev --verbose run test",
+    "test:e2e": "yarn cypress-setup && start-server-and-test 'npm run dev --prefix e2e-projects/cypress-next-app' http://localhost:3000 'SM_ENV=development ENABLE_SENTRY=false npm run slicemachine --prefix e2e-projects/cypress-next-app' http://localhost:9999 'cypress run'",
+    "test:e2e:dev": "yarn cypress-setup && start-server-and-test 'npm run dev --prefix e2e-projects/cypress-next-app' http://localhost:3000 'SM_ENV=development ENABLE_SENTRY=false npm run slicemachine --prefix e2e-projects/cypress-next-app' http://localhost:9999 'cypress open'",
```

Comment:
@mdeclercq remember we had to remove this `SM_ENV=development` to make it work. 
But I discovered that without this Segment is sent to [Prod] SliceMachine and it's not good right?